LICENSES_bundled.txt

```diff
@@ -15,3 +15,8 @@ Name: dragon4
 Files: numpy/core/src/multiarray/dragon4.c
 License: MIT
   For license text, see numpy/core/src/multiarray/dragon4.c
+
+Name: libdivide
+Files: numpy/core/include/numpy/libdivide/*
+License: zlib
```

Comment:
```suggestion
License: Zlib
```
Don't think it matters, but just checked [the list](https://spdx.github.io/spdx-spec/appendix-I-SPDX-license-list/) and maybe it is typically capitalized.